modules/ROOT/pages/index.adoc

```diff
@@ -5,75 +5,46 @@ endif::[]
 :imagesdir: ../assets/images/
 :page-aliases: general:getting-started:anypoint-exchange.adoc, ex-ref-raml.adoc
 
-Anypoint Exchange provides the benefit of being able to discover, share, and incorporate assets and resources into your applications. Anypoint Exchange helps you create API developer portals, view and test APIs, simulate data to APIs (Mocking Service), create assets, and use API Notebooks to describe and test API functions.
+Anypoint Exchange is a curated catalog of reusable assets such as APIs, policies, API spec fragments, API groups, templates, examples, custom assets, and integration assets such as connectors. You can catalog (publish), share, discover, learn about, and reuse assets within your organization to facilitate collaboration, boost productivity, and promote standards.
+
+image:ex2-old-search-landing.png[Exchange landing page]
+
+Use Exchange to discover, share, and incorporate assets and resources into your applications. Exchange helps you create API developer portals, view and test APIs, simulate sending data to APIs by using the mocking service, create assets, and use API Notebooks to describe and test API functions.
+
+Exchange assets are accessible in Anypoint Studio (connectors, templates, and examples), API Manager (APIs, API groups, and policies), and Runtime Manager (applications), and you can publish APIs from Design Center to Exchange.
```

Comment:
I wouldn't enumerate here since it's not an exhaustive list, assets are also accessible in other ways such as Graph Service, Maven Facade, Design Center API Designer